src/config/hardware.c

```diff
@@ -49,6 +49,7 @@ const char * const MODULE_NAME[TX_MODULE_LAST] = {
       [CC2500]   = "CC2500",
       [NRF24L01] = "NRF24l01",
       [MULTIMOD] = "MultiMod",
+      [RADIOLINK_CC2530] = "Radiolink",
```

Comment:
**RadioLink** is their branding.

Reply:
I'll remove it, it's required only if the module is configured from hardware.ini, but I think it'll be hardcoded ?